SampleApplications/Workshop/Aggregation/Server/AggregationNodeManager.cs

```diff
@@ -50,26 +52,36 @@ public class AggregationNodeManager : CustomNodeManager2
         /// <summary>
         /// Initializes the node manager.
         /// </summary>
-        public AggregationNodeManager(IServerInternal server, ApplicationConfiguration configuration, ConfiguredEndpoint endpoint, bool ownsTypeModel)
+        public AggregationNodeManager(
+            IServerInternal server,
+            ApplicationConfiguration configuration,
+            ConfiguredEndpoint endpoint,
+            Opc.Ua.Client.ReverseConnectManager reverseConnectManager,
+            bool ownsTypeModel)
         :
             base(server, configuration, Namespaces.Aggregation, AggregationModel.Namespaces.Aggregation)
         {
             SystemContext.NodeIdFactory = this;
 
             m_configuration = configuration;
             m_endpoint = endpoint;
+            if (endpoint.ReverseConnect != null &&
+                endpoint.ReverseConnect.Enabled)
```

Comment:
In this case you can check if reverseConnectManager is null and throw a ArgumentNullException

Reply:
indeed...

---
In reply to: [432435831](https://github.com/OPCFoundation/UA-.NETStandard/pull/998#discussion_r432435831) [](ancestors = 432435831)